6/Dockerfile

```diff
@@ -29,14 +37,20 @@ LABEL summary="$SUMMARY" \
       io.k8s.display-name="Node.js $NODEJS_VERSION" \
       io.openshift.expose-services="8080:http" \
       io.openshift.tags="builder,nodejs,nodejs$NODEJS_VERSION" \
+      io.openshift.s2i.scripts-url="image:///usr/local/s2i" \
```

Comment:
Defined as `image:///usr/libexec/s2i` in https://github.com/sclorg/s2i-base-container/blob/master/core/Dockerfile

Is there any reason to change this?